app/src/main/java/com/example/solidarity/Event.java

```diff
@@ -70,6 +70,34 @@ public void setEventDate(Date date) {
         put(KEY_DATE, date);
     }
 
+    public static String getKeyDescription() {
```

Comment:
It's actually better to use KEY_DESCRIPTION directly rather than having a getter because it is a constant. Not sure if you have seen my other comment or not in a earlier PR